homeassistant/components/googlehome/sensor.py

```diff
@@ -35,11 +34,16 @@
     if discovery_info is None:
         _LOGGER.warning(
             "To use this you need to configure the 'googlehome' component")
+        return
+
+    await hass.data[CLIENT].update_info(discovery_info['host'])
 
     devices = []
     for condition in SENSOR_TYPES:
+        data = hass.data[GOOGLEHOME_DOMAIN][discovery_info['host']]
+        info = data.get('info', {})
```

Comment:
These two lines can be moved too, right?

Reply:
yes, my bad

Reply:
Should be fine now 👍